test/docs-sane

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env python
```

Comment:
We should probably "versionize" this in in our spec file, e.g., `%{versionize_script python3 test/docs-sane}`